src/backend/utils/storage.js

```diff
@@ -53,11 +56,19 @@ module.exports = {
 
   setInvalidFeed: (id, reason) => {
     const key = createInvalidFeedKey(id);
-    redis.set(key, reason);
+    return redis.set(key, reason);
   },
 
   isInvalid: id => redis.exists(createInvalidFeedKey(id)),
 
+  setDelayedFeed: async (id, seconds) => {
+    const key = createDelayedFeedKey(id);
+    const msg = await redis.set(key, seconds, 1);
+    return msg === 'OK';
```

Comment:
I don't think we care about this one, let's just return the promise and let it fail that way.

```js
setDelayedFeed: (id, seconds) => redis.set(createDelayedFeedKey(id), seconds, 1);
```

Reply:
Done! https://github.com/Seneca-CDOT/telescope/pull/726/commits/7f528072e0d5b62e30c4dc0ec9e3b7fd7f35afbf